.github/workflows/test.yml

```diff
@@ -61,16 +71,20 @@ jobs:
           export LD_LIBRARY_PATH=${R_HOME}/lib
           cargo test -- --nocapture --test-threads=1
 
+
       - name: Build (Windows)
         if: runner.os == 'Windows'
         run: cargo build --target x86_64-pc-windows-gnu
-
-## Tests don't currently succeed on Windows
-## The following test hangs: tests::r_output_test
-## See: https://github.com/extendr/extendr/issues/67
-#      - name: Run tests (Windows)
-#        if: runner.os == 'Windows'
-#        run: cargo test -- --nocapture --test-threads=1
+        env:
+          LIBCLANG_PATH: C:\msys64\mingw64\bin
+        shell: msys2
```

Comment:
The syntax is `msys2 {0}` for some reason.

Reply:
See e.g. here: https://github.com/extendr/libR-sys/blob/6ff5c6c496ed38143599bd1801b2c7999c765173/.github/workflows/test.yml#L79

Reply:
Yes I saw that; new commit incoming. The project builds fine on my local machine, I hope it is indeed `msys2` issue.